### DISCUSSION TITLE
add count of columns for the results in the status bar

### DISCUSSION THREAD

**Discussion by abusen333** [DaxStudio/DaxStudio#1139] [Ideas]
Showing the count for columns in the status bar is helpful for reproducibility of DAX query. Sometimes I want to check if the new query has the same count of columns as the old ones. I haven't find any DAX function returns the count of columns for a given query so I have to do it manually.

**Comment by dgosbell** [comment]
I don't think there are any functions to return a count of the columns for a DAX expression, so I don't think there is any easy way to do this yourself. But it should not be too hard to add this to the status bar so I'll add an issue to track this request
